docs/reference/configuration.md

```diff
@@ -183,7 +183,7 @@ type = 'File'
 data_dir = '/tmp/greptimedb/data/'
 
 [meta_client_opts]
-metasrv_addr = "1.1.1.1:3002"
+metasrv_addrs = ["1.1.1.1:3002"]
```

Comment:
Change it to `127.0.0.1:3002`

Reply:
@waynexia Just a reminder in case you did not see this comment

Reply:
> @waynexia Just a reminder in case you did not see this comment

I've already resolved the comment.